src/inmanta/data/__init__.py

```diff
@@ -5267,6 +5267,59 @@ def to_dto(self) -> m.Notification:
         )
 
 
+class EnvironmentMetricsCounter(BaseDocument):
+    """
+    A metric that is a counter
+
+    :param metric_name: The name of the metric
+    :param timestamp: The timestamps at which a new record is created
+    :param count: the counter for the metric for the given timestamp
+    """
+
+    metric_name: str
+    timestamp: datetime.datetime
+    count: int
+
+    __primary_key__ = ("metric_name", "timestamp")
+
+    @classmethod
+    def new(cls, metric_name: str, timestamp: datetime.datetime, count: int) -> "EnvironmentMetricsCounter":
```

Comment:
What is this for?

Reply:
it is used here: https://github.com/inmanta/inmanta-core/pull/5198/files#diff-8abb50042fc34e38c278ec05a3fc85edfdb092058a78b62a8a2796ddbd6bb708R110

Reply:
You can just create an instance of EnvironmentMetricsCounter using keyword arguments. For example:

```
EnvironmentMetricsCounter(metric_name="test", timestamp=datetime.datetime.now(), count=1)
```